server/src/test/java/org/elasticsearch/cluster/metadata/ToAndFromJsonMetaDataTests.java

```diff
@@ -111,7 +111,7 @@ public void testSimpleJsonFromAndTo() throws IOException {
                         .putMapping("mapping1", MAPPING_SOURCE1)
                         .putMapping("mapping2", MAPPING_SOURCE2)
                         .putAlias(newAliasMetaDataBuilder("alias1").filter(ALIAS_FILTER1))
-                        .putAlias(newAliasMetaDataBuilder("alias2"))
+                        .putAlias(newAliasMetaDataBuilder("alias2").writeIndex(true))
```

Comment:
some randomization comments

Reply:
as before, I chose to leave the static nature of this test. I can update here